docs/dictionary/command/decrypt-using-rsa.lcdoc

```diff
@@ -36,12 +37,12 @@ Use the <decrypt using rsa> command to decrypt a message using RSA
 public key encryption.
 
 Use the form
-decrypt <message> with private key <key> 
+'decrypt <message> with private key <key> '
```

Comment:
This should have an empty line above and below and be indented 4 spaces so it is code formatted. Perhaps a colon after form too.